include/deal.II/grid/tria_accessor.h

```diff
@@ -2110,6 +2110,17 @@ class TriaAccessor<0, 1, spacedim>
   types::boundary_id boundary_id () const;
 
   /**
+   * Return a constant reference to the manifold object used for this object.
+   *
+   * As explained in
+   * @ref boundary "Boundary and manifold description for triangulations",
+   * the process involved in finding the appropriate manifold description
+   * involves querying both the manifold or boundary indicators. See there for
+   * more information.
```

Comment:
but you're not actually querying anything other than the manifold_id in the implementation. is this text out of date?